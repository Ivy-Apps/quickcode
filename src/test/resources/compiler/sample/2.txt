fun main() {
    println("Hello, {{name}}!")
#if {{morning}} #then
    println("Good morning :)")
#endif
}