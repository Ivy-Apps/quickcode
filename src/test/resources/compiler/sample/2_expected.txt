fun main() {
    println("Hello, QuickCode!")
    println("Good morning :)")
}